docs/installation.rst

```diff
@@ -0,0 +1,217 @@
+Installation and Setup
+======================
+
+An overview of common installation and setup steps when working with Runhouse.
+
+- Installing the Runhouse library
+- Cloud providers and credentials
+- Cluster lifecycle management
+- Authenticate with Runhouse Den
+
+Install the Runhouse library
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Optionally, to begin, set up a virtual environment with `Conda <https://conda.io/projects/conda/en/latest/user-guide/install/index.html>`.
```

Comment:
Conda is one type of virtual env that a user might use,  can we make it more clear that it isn't the only path?

Reply:
Agreed, I'll update to clarify

Reply:
Does the updated version seem better?